frontend/src/lib/utils/navigation/route.ts

```diff
@@ -2,37 +2,73 @@
  * The allowed routes
  */
 
-export enum Route {
-  About = 'about',
-  CandAppFAQ = 'candidate/faq',
-  CandAppFeedback = 'candidate/feedback',
-  CandAppForgotPassword = 'candidate/forgot-password',
-  CandAppHelp = 'candidate/help',
-  CandAppHome = 'candidate',
-  CandAppInfo = 'candidate/info',
-  CandAppPreview = 'candidate/preview',
-  CandAppProfile = 'candidate/profile',
-  CandAppQuestions = 'candidate/questions',
-  CandAppQuestionEdit = 'candidate/questions/edit',
-  CandAppRegister = 'candidate/register',
-  CandAppResetPassword = 'candidate/password-reset',
-  CandAppSettings = 'candidate/settings',
-  Candidate = 'candidates',
-  Candidates = 'candidates',
+export type RouteOptions =
+  | 'About'
+  | 'CandAppFAQ'
+  | 'CandAppFeedback'
+  | 'CandAppForgotPassword'
+  | 'CandAppHelp'
+  | 'CandAppHome'
+  | 'CandAppInfo'
+  | 'CandAppPreview'
+  | 'CandAppProfile'
+  | 'CandAppQuestions'
+  | 'CandAppQuestionEdit'
+  | 'CandAppRegister'
+  | 'CandAppResetPassword'
+  | 'CandAppSettings'
+  | 'Candidate'
+  | 'Candidates'
   /** The Help route is currently redirected to About */
-  Help = 'about',
-  Home = '',
-  Info = 'info',
-  Intro = 'intro',
-  Parties = 'parties',
-  Party = 'parties',
-  Privacy = 'privacy',
-  Question = 'questions',
-  QuestionCategory = 'questions/category',
-  Questions = 'questions',
-  ResultCandidate = 'results/candidate',
-  ResultParty = 'results/party',
-  Results = 'results',
-  Statistics = 'results/statistics',
-  _Test = '_test'
-}
+  | 'Help'
+  | 'Home'
+  | 'Info'
+  | 'Intro'
+  | 'Parties'
+  | 'Party'
+  | 'Privacy'
+  | 'Question'
+  | 'QuestionCategory'
+  | 'Questions'
+  | 'ResultCandidate'
+  | 'ResultParty'
+  | 'Results'
+  | 'Statistics'
+  | '_Test';
+
+export const Route: Record<RouteOptions, string> = {
```

Comment:
Could you use `CAMEL_CASE` for all `consts` while reserving `CamelCase` for the associated types? I.e. this would become `const ROUTE` and the type below just `type Route`.